Susi/MainViewController.swift

```diff
@@ -136,10 +139,16 @@ class MainViewController: UICollectionViewController, UICollectionViewDelegateFl
     
     // Setup Collection View
     func setupCollectionView() {
+        // Check if user defaults have an image, set background as image
+        if let userDefaultData = getWallpaperFromUserDefaults() {
+            if let imageData = userDefaultData as? Data { // Check if object saved in user defaults if of type Data
+                setBackgroundImage(image: UIImage(data : imageData))
+            }
+        }
         collectionView?.backgroundColor = .clear
         collectionView?.delegate = self
 
-        collectionView?.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height - 50)
+        collectionView?.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height - 45)
```

Comment:
Since you kept collection view background as transparent , you could not see that their was a small gap between chat view and input options , adjusted the value and check on iphone 5s,6,6s and 7 series to confirm the change was necessary